backend/src/main/java/com/tourneynizer/tourneynizer/controller/UserController.java

```diff
@@ -37,4 +36,32 @@ public UserController(UserService userService) {
 
         return new ResponseEntity<>(user, new HttpHeaders(), HttpStatus.OK);
     }
+
```

Comment:
For now until we do email verification do you want to send cookies when creating the user object is created?